docs/editors/vscode.md

```diff
@@ -271,3 +270,7 @@ extension to use default IntelliJ shortcuts with VS Code.
 | Type info        | Show hover                |
 | Expand           | Fold                      |
 | Extend Selection | Expand selection          |
+
+## GitHub Codespaces and GitHub.dev support
+
+See https://scalameta.org/metals/docs/editors/online-ides#github-codespaces-and-githubdev
```

Comment:
the domain is hardcoded, since we copy-paste this file to the VS Code README during the release process